app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    splitted_input = command.split()
+    if (len(splitted_input) == 3
+            and splitted_input[0] == "cp"
+            and splitted_input[1] != splitted_input[2]):
```

Comment:
It would be better to use unpacking instead of retrieving values by index in the future in such cases to improve code readability.